jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -195,6 +212,9 @@ def _on_display_units_changed(self, event={}):
                     new_unit = u.Unit(self.display_unit)
 
                     bg = self.background_value * prev_unit
+
+                    # will need to add additonal custom equiv here once
+                    # pix2<>angle is enabled

```

Comment:
can you add a link back here from that ticket please?

Reply:
this would go away if we pass all conversions through utils.flux_conversion